Misc/NEWS.d/next/Library/2024-05-28-14-35-23.gh-issue-97850.dCtjel.rst

```diff
@@ -0,0 +1 @@
+Remove :func:`!pkgutil.get_loader` and :func:`!pkgutil.find_loader`.
```

Comment:
How about "Remove ... , deprecated since Python 3.12"?  I don't actually remember how we typically state these deprecated removals.

Reply:
I will check the formulation but it's better to add "deprecated", yes!